### ISSUE TITLE
Drop php-mf2

### ISSUE THREAD

**Issue by janboddez** [janboddez/addon-for-activitypub#26]
We can likely use `WP_HTML_Tag_Processor` instead.

**Comment by janboddez** [comment]
We _could_, but there are scenarios (like when you have an h-card with a u-url, all inside a h-cite, before the u-url of the linked page we actually want to find) in which doing so could fail.

What we could do is look for IndieBlocks' blocks, and only parse HTML when they're present. This HTML is more likely to follow IndieBlocks' output. Caveat: This, too would fail, if one used, e.g., a custom HTML block for reply context, or converted one of IndieBlocks' blocks to Custom HTML (or a Group, even, because, well, IndieBlocks behaves a bit funny).

In other words, we may want to keep php-mf2 around for now. IndieBlocks uses it too, and also parses posts (to look for likes or bookmarks, which may get a "different" automatically generated title, depending on one's settings); the result is cached either way, so the overhead should be negligible.
